Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -0,0 +1,297 @@
+            set
+            {
+                customHandMaterial = value;
+                OnCustomHandMaterialUpdate?.Invoke();
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("If true, will update material pinch strength using OVR Values.")]
+        private bool updateMaterialPinchStrengthValue = true;
+
+        /// <summary>
+        /// If true, will update material pinch strength using OVR Values.
+        /// </summary>
+        public bool UpdateMaterialPinchStrengthValue => UseCustomHandMaterial && updateMaterialPinchStrengthValue;
+
+        [SerializeField]
+        [Tooltip("Property in custom material used to visualize pinch strength.")]
+        private string pinchStrengthMaterialProperty = "_PressIntensity";
+
+        /// <summary>
+        /// Property in custom material used to visualize pinch strength.
+        /// </summary>
+        public string PinchStrengthMaterialProperty => pinchStrengthMaterialProperty;
+
+#if OCULUSINTEGRATION_PRESENT
+        [Header("Hand Tracking Configuration")]
+        [SerializeField]
+        [Tooltip("Setting this to low means hands will continue to track with low confidence.")]
+        private OVRHand.TrackingConfidence _minimumHandConfidence = OVRHand.TrackingConfidence.Low;
+
+        /// <summary>
+        /// Setting this to low means hands will continue to track with low confidence.
+        /// </summary>
+        public OVRHand.TrackingConfidence MinimumHandConfidence
+        {
+            get => _minimumHandConfidence;
+            set => _minimumHandConfidence = value;
+        }
+
+        /// <summary>
+        /// Current tracking confidence of left hand. Value managed by OculusQuestHand.cs.
+        /// </summary>
+        public OVRHand.TrackingConfidence CurrentLeftHandTrackingConfidence { get; set; }
+
+        /// <summary>
+        /// Current tracking confidence of right hand. Value managed by OculusQuestHand.cs.
+        /// </summary>
+        public OVRHand.TrackingConfidence CurrentRightHandTrackingConfidence { get; set; }
+#endif
+
+        [SerializeField]
+        [Range(0f, 5f)]
+        [Tooltip("Time after which low confidence is considered unreliable, and tracking is set to false. Setting this to 0 means low-confidence is always acceptable.")]
+        private float _lowConfidenceTimeThreshold = 0.2f;
+
+        /// <summary>
+        /// Time after which low confidence is considered unreliable, and tracking is set to false.
+        /// </summary>
+        public float LowConfidenceTimeThreshold
+        {
+            get => _lowConfidenceTimeThreshold;
+            set => _lowConfidenceTimeThreshold = value;
+        }
+
+        [Header("Performance Configuration")]
+        [SerializeField]
+        [Tooltip("Default CPU performance level (0-2 is documented), (3-5 is undocumented).")]
+        [Range(0, 5)]
+        private int defaultCpuLevel = 2;
+
+        /// <summary>
+        /// Accessor for the Oculus CPU power level.
```

Comment:
Nit: Add a link to:

https://developer.oculus.com/documentation/native/android/mobile-power-overview

Which is what I'm guessing this is referring to.